lib/axon/losses.ex

```diff
@@ -1024,4 +1024,175 @@ defmodule Axon.Losses do
       end
     )
   end
+
+  @doc """
+  Connectionist Temporal Classification loss.
+
+  ## Argument Shapes
+
+    * `l_true` - $\(B\)$
+    * `y_true` - $\(B, S\)$
+    * `y_pred` - $\(B, T, D\)$
+
+  ## Options
+
+  * `:reduction` - reduction mode. One of `:sum` or `:none`.
+    Defaults to `:none`.
+
+  ## Description
+    `l_true` contains lengths of target sequences. Nonzero positive values.
+    `y_true` contains target sequences. Each value represents a class
+    of element in range of available classes 0 <= y < D. Blank element
+    class is included in this range, but shouldn't be presented among
+    y_true values. Maximim target sequence length should be lower or equal
+    to `y_pred` sequence length: S <= T.
+    `y_pred` - logarithmized probabilities of classes D along the
```

Comment:
I think it's okay to just call this log probabilities since that's pretty common verbage

```suggestion
    `y_pred` - log probabilities of classes D along the
```

Reply:
It's how they like to call it in pytorch docs.